### PR TITLE
Fix/flash deconv updates

### PR DESCRIPTION
**Pull request by poshul**
## Checklist
- [ ] Make sure that you are listed in the AUTHORS file
- [ ] Add relevant changes and new features to the CHANGELOG file
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] New and existing unit tests pass locally with my changes
- [ ] Updated or added python bindings for changed or new classes (Tick if no updates were necessary.)

### How can I get additional information on failed tests during CI
<details>
  <summary>Click to expand</summary>
If your PR is failing you can check out

- The details of the action statuses at the end of the PR or the "Checks" tab.
- http://cdash.openms.de/index.php?project=OpenMS and look for your PR. Use the "Show filters" capability on the top right to search for your PR number.
  If you click in the column that lists the failed tests you will get detailed error messages.

</details>

### Advanced commands (admins / reviewer only)
<details>
  <summary>Click to expand</summary>
  
- `/reformat` (experimental) applies the clang-format style changes as additional commit. Note: your branch must have a different name (e.g., yourrepo:feature/XYZ) than the receiving branch (e.g., OpenMS:develop). Otherwise, reformat fails to push.
- setting the label "NoJenkins" will skip tests for this PR on jenkins (saves resources e.g., on edits that do not affect tests)
- commenting with `rebuild jenkins` will retrigger Jenkins-based CI builds
  
</details>

---
:warning: Note: Once you opened a PR try to minimize the number of *pushes* to it as every push will trigger CI (automated builds and test) and is rather heavy on our infrastructure (e.g., if several pushes per day are performed).

### CODE REVIEW

File: src/openms/source/ANALYSIS/TOPDOWN/QScore.cpp [hunk 1]
```diff
@@ -47,17 +47,8 @@ namespace OpenMS
...
 
-    //ChargeCos         1.492
-    //ChargeSNR       -2.0041
-    //Cos            -14.3891
-    //SNR             -0.9853
-    //ChargeScore      0.4568
-    //AvgPPMerror       0.063
-    //Intercept       14.4072
-
     // ChargeCos           0.0941
```
Comment (timosachsenberg) [inline src/openms/source/ANALYSIS/TOPDOWN/QScore.cpp:52]: these numbers don't match the ones in the weights (e.g. sign)

Comment (timosachsenberg) [reply src/openms/source/ANALYSIS/TOPDOWN/QScore.cpp:52]: please double check... this could be a bug

Comment (JeeH-K) [reply src/openms/source/ANALYSIS/TOPDOWN/QScore.cpp:52]: @KyowonJeong would it be okay to delete these lines?

Comment (timosachsenberg) [reply src/openms/source/ANALYSIS/TOPDOWN/QScore.cpp:52]: note that the sign of cos is negative in the weights vector but positive in the comment.
on the other hand the sign of chargecos is positive in the weights vector but also positive in the comment.

File: src/openms/include/OpenMS/ANALYSIS/TOPDOWN/PeakGroup.h [hunk 2]
```diff
@@ -208,10 +208,10 @@ namespace OpenMS
     bool isTargeted() const;
 
     /// get the decoy flag of this
-    int getDecoyFlag() const;
+    decoyFlag getDecoyFlag() const;
```
Comment (timosachsenberg) [inline src/openms/include/OpenMS/ANALYSIS/TOPDOWN/PeakGroup.h:211]: minor thing: type looks like a method with this capitalization
either: DecoyFlag or DECOY_FLAG are styles we currently have for enums in our code base

File: src/openms/source/ANALYSIS/TOPDOWN/QScore.cpp (hunk 1 above)
Comment (KyowonJeong) [reply src/openms/source/ANALYSIS/TOPDOWN/QScore.cpp:52]: Removed. THanks!

**Pull request closed without merging**